content/articles/deploying-a-serverless-django-restapi-with-zappa-on-aws/index.md

```diff
@@ -0,0 +1,449 @@
+
+Briefly, let us set up AWS on your console to properly utilize Zappa.
+
+S*tep 1*: Log in to your AWS console and go to the IAM section
+
+*Step 2*: On the left side of the screen click on *users*
+
+*Step 3*: Click on *add users*. For this project, we will name our 'user', *'serverless'* and for access type, choose '*Programmatic access*'. This uses the API to interact with AWS.
+
+*Step 4*: Move on the permission and select *'Attach existing policies directly*. We will be accessing two policies, the first is '*IAMFullAccess*" This creates users for lambda to be executed. The second is *'PowerUserAccess'* for creating API gateway and S3 bucket. Search for these policies and click on their boxes.
+
+*Step 5*: Skip Add tags, move to Review and create a user.
+
+*Successfully created!* We have also been supplied with an access key and secret access key. Download the *.csv file* which contains these keys as these pieces of information will be lost when the page closes
+
+Briefly, let'sSet up AWS on your console in other to properly utilize Zappa. 
+
+### Deploying with Zappa
+
+The first step is to install Zappa in our virtual environment:
+
+```python
+pip install zappa
+```
+
+Next, initialize Zappa, 
+
+```python
+zapa init
+```
+
+```python
+███████╗ █████╗ ██████╗ ██████╗  █████╗
+╚══███╔╝██╔══██╗██╔══██╗██╔══██╗██╔══██╗
+  ███╔╝ ███████║██████╔╝██████╔╝███████║
+ ███╔╝  ██╔══██║██╔═══╝ ██╔═══╝ ██╔══██║
+███████╗██║  ██║██║     ██║     ██║  ██║
+╚══════╝╚═╝  ╚═╝╚═╝     ╚═╝     ╚═╝  ╚═╝
+
+Welcome to Zappa!
+
+Zappa is a system for running server-less Python web applications on AWS Lambda and AWS API Gateway.
+This `init` command will help you create and configure your new Zappa deployment.
+Let's get started!
+
+Your Zappa configuration can support multiple production stages, like 'dev', 'staging', and 'production'.
+What do you want to call this environment (default 'dev'): dev
+```
+
+Flowing from this instruction, select an environment name. In this project, we used the default name, '*dev'.*
+![Zappa initalisation](/engineering-education/deploying-a-serverless-django-restapi-with-zappa-on-aws/zappainit.PNG)
+
+```python
+It looks like this is a Django application!
+What is the module path to your projects's Django settings?
+We discovered: bookstore.settings
+Would you like to deploy this application ←[1mglobally←[0m? (default 'n') [y/n/(p)rimary]: n
+
+Okay, here's your zappa_settings.json:
+
+{
+    "dev": {
+        "aws_region": "eu-west-2",
+        "django_settings": "bookshop.settings",
+        "profile_name": "default",
+        "project_name": "bookstore",
+        "runtime": "python3.9",
+        "s3_bucket": "zappa-763nx6mgt"
+    }
+}
+```

```

Comment:
I would suggest to make these code blocks as images, since as you see here, the characters are getting encoded to some other format, which is not markdown.